src/test/java/redis/clients/jedis/csc/JedisPooledClientSideCacheTest.java

```diff
@@ -1,13 +1,21 @@
 package redis.clients.jedis.csc;
 
+import io.redis.test.annotations.SinceRedisVersion;
+import io.redis.test.utils.RedisVersionRule;
 import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import redis.clients.jedis.HostAndPorts;
 
+@SinceRedisVersion(value = "7.4.0", message = "Jedis client-side caching is only supported with Redis 7.4 or later.")
 public class JedisPooledClientSideCacheTest extends JedisPooledClientSideCacheTestBase {
 
+  @ClassRule
+  public static RedisVersionRule versionRule = new RedisVersionRule(
+          HostAndPorts.getRedisEndpoint("standalone1").getHostAndPort(),
+          HostAndPorts.getRedisEndpoint("standalone1").getClientConfigBuilder().build());
```

Comment:
You've already added the shortcut.
```suggestion
  public static RedisVersionRule versionRule = new RedisVersionRule(
          HostAndPorts.getRedisEndpoint("standalone1"));
```